app/models/course.rb

```diff
@@ -1,4 +1,8 @@
 class Course < ActiveRecord::Base
+  enum locale: {
+      en_US: 0,
+      hi_IN: 1
+  }
```

Comment:
use same constant enum locale in all the occurrence, 
if not that atleast keep the enums with same values across the files

Reply:
Need to discuss about updating migration files as well.